src/core_landice/Registry.xml

```diff
@@ -567,6 +603,14 @@
             <var name="uReconstructX" packages="higherOrderVelocity"/>
 			<var name="uReconstructY" packages="higherOrderVelocity"/>
                         <var name="basalFrictionFlux"/>
+			<var name="damage"/>
+			<var name="ddamagedt"/>
+			<var name="szero"/>
```

Comment:
maybe more clear to call this "snaught" or "s0" ?

Reply:
now I use s0